src/drivers/cmakeDriver.ts

```diff
@@ -1084,6 +1084,12 @@ export abstract class CMakeDriver implements vscode.Disposable {
             versionOutputRegexp: "version ([^\\s]+)",
             captureGroup: 1
         },
+        {
+            name: "chesscc",
```

Comment:
does this require any other changes for the compiler to be used properly? Has this been tested?

Reply:
nvm I see that you couldn't find the install to test

Reply:
Couldn't test because I didn't find the compiler itself anywhere to install but 99%, a "chess**cc**" compiler would behave like **cc**, so I think the regexp (the same as for cc) is the right one.